### PR TITLE
Fix recursive filtering for Loader

### PR DESCRIPTION
**Pull request by kaamaurice**
## Brief description
Since OP 3.15.0 Loader doesn't filter asset list recursively

### CODE REVIEW

**Review by Sharkitty — APPROVED** [review]
LGTM

**Comment by Tilix4** [comment]
Isn't it something that must go into OP's core code?

File: openpype/tools/utils/models.py [hunk 1]
```diff
@@ -234,6 +234,11 @@ def filterAcceptsRow(self, row, parent_index):
             row, parent_index
         )
 
+    def __init__(self, *args, **kwargs):
```
Comment (Tilix4) [inline openpype/tools/utils/models.py:237]: `__init__` must be as close to the class name as possible.

**Review by Tilix4 — CHANGES_REQUESTED** [review]

**Comment by Tilix4** [comment]
Created PR to `develop`: https://github.com/ynput/OpenPype/pull/4595

**Pull request closed without merging**